pkg/nsx/services/inventory/build.go

```diff
@@ -172,3 +172,32 @@ func (s *InventoryService) compareAndMergeUpdate(pre interface{}, cur interface{
 		return operationNone, nil
 	}
 }
+
+func (s *InventoryService) BuildNamespace(namespace *corev1.Namespace) (retry bool) {
+	log.Info("Building Namespace ", "Namespace", namespace.Name)
+	retry = false
+
+	object := s.ProjectStore.GetByKey(string(namespace.UID))
+	var preContainerProject containerinventory.ContainerProject
+	if object != nil {
+		preContainerProject = *object.(*containerinventory.ContainerProject)
+	}
+
```

Comment:
You may change it to:
preContainerProject := s.ProjectStore.GetByKey(string(namespace.UID))
if preContainerProject != nil {
		preContainerProject = *object.(*containerinventory.ContainerProject)
	}

Reply:
ok